CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Features
 
+[jest-core] Enable testResultsProcessor to be async (#13343)
```

Comment:
Needs the actual link as well, not just the number (see other entries

Reply:
Got it 👍 Updated